find-the-middle-element/solution.js

```diff
@@ -0,0 +1,8 @@
+var gimme = function (inputArray) {
+  const originalArray = inputArray.slice(0);
+  const sortedArray = inputArray.sort(function(a,b) {
```

Comment:
Since this is a single line function that returns something, it's a great opportunity to use an arrow function instead: I'd write this as `inputArray.sort( (a,b) => a-b);